lib/callbacks.js

```diff
@@ -15,12 +15,15 @@ module.exports = function(request) {
 
       if (!res.body || !Object.keys(res.body).length)
         console.error(res.text);
-
-      if (res.text)
+      if (res.text === JSON.stringify(res.body)){
         console.log(res.text);
```

Comment:
a little pedantic but if the two are the same I'd prefer to log the `res.body` rather than the `res.text`